src/objects/Project__c.object

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't seen any question that mentions creating a new object. Could you please verify?